adobe/spectrum-web-components
# docs: build storybook in preview mode and link to it correctly everywhere

## Description
- build storybook in preview sites
- link to storybook correctly from subpages/directories
- scope SW correctly

## Motivation and context
Storybook should be available on previews and the service worker should work in production

## How has this been tested?

    1. Go to https://sw-updates--spectrum-web-components.netlify.app/components/button/
    2. Click the "Playground" link in the side nav
    3. See that you made it to Storybook

## Types of changes

## Checklist
